### PR TITLE
Option to select the required DDP package

### PR DESCRIPTION
**Pull request by Jaganathancse**
This changes to provide the option to choose the required DDP package when multiple different packages are available.

### CODE REVIEW

**Review by rebtoor — CHANGES_REQUESTED** [review]
I'd prefer to have a role with all these tasks integrated in an existing playbook or in a new one. It would give us more flexibility in using it.

Strictly speaking about the code, you should use FQCN for all used ansible modules.

**Review by jpodivin — CHANGES_REQUESTED** [review]
I believe `edpm_module_load` would be useful in this case. Also it may be for the best to turn other tasks into separate role of it's own, with properly exposed parameters, I assume similar installation procedure could be useful for other Intel drivers?

File: playbooks/deploy_edpm_kernel_ddp_pkg.yaml
Comment (slagle) [inline playbooks/deploy_edpm_kernel_ddp_pkg.yaml:1]: I agree with the other comments that these tasks need to be added to an existing role, or a new role as opposed to a new playbook. EDPM consumes the roles, not the playbooks.

**Review by slagle — CHANGES_REQUESTED** [review]

File: playbooks/deploy_edpm_kernel_ddp_pkg.yaml
Comment (Jaganathancse) [reply playbooks/deploy_edpm_kernel_ddp_pkg.yaml:1]: Ack

**Review by rebtoor — APPROVED** [review]

**Pull request merged**